### PR TITLE
docs: add tutorial 

### LINKED ISSUE #80: Write saiph tutorial
The tutorial is empty (https://github.com/octopize/saiph/blob/main/docs/tutorial.rst)

### PR DESCRIPTION
**Pull request by jpetot**
Fix #80

### CODE REVIEW

**Review by tcrasset — APPROVED** [review]
Nice!

File: docs/example/README.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Tutorial
+================
+
+Hello you
```
Comment (olivierabz) [inline docs/example/README.rst:4]: Is this wanted?

**Review by olivierabz — APPROVED** [review]
Looks good

File: docs/example/README.rst (hunk 1 above)
Comment (jpetot) [reply docs/example/README.rst:4]: nothing special, a readme is need for sphinx-gallery

**Pull request merged**